### PR TITLE
GEOPY-1531: r2

### PR DESCRIPTION
**Pull request by domfournier**
**GEOPY-1531 - Implement rescaling of misfit functions for joint inversion**
Add directive to auto scale the joint problems

### CODE REVIEW

File: simpeg/directives/directives.py [hunk 1]
```diff
@@ -3426,3 +3426,70 @@ def endIter(self):
...
+        with open(self.filepath, "w", encoding="utf-8") as f:
+            f.write(
+                "Iterations\t"
+                + "\t".join(
+                    f"[{objfct.name}]" for objfct in self.invProb.dmisfit.objfcts
+                )
+            )
+            f.write("\n")
+
+    def endIter(self):
```
Comment (benk-mira) [inline simpeg/directives/directives.py:3459]: snake case?

File: simpeg/directives/directives.py [hunk 2]
```diff
@@ -3426,3 +3426,70 @@ def endIter(self):
             for directive in directiveList:
                 if not isinstance(directive, SaveIterationsGeoH5):
                     directive.endIter()
+
+
+class ScaleMisfitMultipliers(InversionDirective):
+    """
+    Scale the misfits by a factor.
```
Comment (benk-mira) [inline simpeg/directives/directives.py:3433]: Can you describe what it is you're scaling the misfit by?  This is only useful for joint?  If so should that be part of the name of the directive?

**Review by benk-mira — CHANGES_REQUESTED** [review]
I'm not familiar with the scaling you're doing, would be nice to beef up the docstring.

File: simpeg/directives/directives.py (hunk 1 above)
Comment (domfournier) [reply simpeg/directives/directives.py:3459]: No, can't change that. Core simpeg

**Review by benk-mira — APPROVED** [review]

**Pull request merged**